### PR TITLE
Tag Atom.jl v0.5.9

### PR DESCRIPTION
**Pull request by attobot**
Repository: [JunoLab/Atom.jl](https://github.com/JunoLab/Atom.jl)
Release: [v0.5.9](https://github.com/JunoLab/Atom.jl/releases/tag/v0.5.9)
Travis: [Travis Build Status](https://travis-ci.org/JunoLab/Atom.jl/builds/201221565)
Diff: [vs v0.5.8](https://github.com/JunoLab/Atom.jl/compare/e7bbd02dccdddadffdd7727d03bf0d5ddc968553...31eb7bb67f322a00a84b11344d53ebb3fba8feae)
`requires` vs v0.5.8: no changes
cc: @MikeInnes

### CODE REVIEW

File: Atom/versions/0.5.9/requires [hunk 1]
```diff
@@ -0,0 +1,13 @@
+julia 0.5-
```
Comment (tkelman) [inline Atom/versions/0.5.9/requires]: drop the trailing dash if you don't need to support prereleases any more

File: Atom/versions/0.5.9/requires [hunk 2]
```diff
@@ -0,0 +1,13 @@
+julia 0.5
+Juno
```
Comment (tkelman) [inline Atom/versions/0.5.9/requires]: `errtrace` and `errmsg` don't exist until 0.2.6

**Comment by MikeInnes** [comment]
0.2.6 isn't a hard requirement – Atom.jl imports the functions but doesn't call them

**Comment by tkelman** [comment]
there are several calls here https://github.com/JunoLab/Atom.jl/blob/31eb7bb67f322a00a84b11344d53ebb3fba8feae/src/display/errors.jl

**Pull request merged**